src/packer.pkr.hcl

```diff
@@ -136,19 +149,240 @@ build {
+  #   # The build will fail if the restart process takes longer than 5 minutes.
+  #   restart_timeout = "5m"
+  # }
+
+  # provisioner "powershell" {
+  #   # Install password packages
+  #   environment_vars = [
+  #     "Category=passwords",
+  #     "DriveLetter=${var.drive_letter}",
+  #     "PackagesDir=${var.packages_dir}"
+  #   ]
+  #   script = "src/powershell/install-category.ps1"
+  # }
+
+  # provisioner "windows-restart" {
+  #   # Wait a maximum of 5 minutes for Windows to restart.
+  #   # The build will fail if the restart process takes longer than 5 minutes.
+  #   restart_timeout = "5m"
+  # }
+
+  # provisioner "powershell" {
+  #   # Install reverse engineering packages
+  #   environment_vars = [
+  #     "Category=reverse-engineering",
+  #     "DriveLetter=${var.drive_letter}",
+  #     "PackagesDir=${var.packages_dir}"
+  #   ]
+  #   script = "src/powershell/install-category.ps1"
+  # }
+
+  # provisioner "windows-restart" {
+  #   # Wait a maximum of 5 minutes for Windows to restart.
+  #   # The build will fail if the restart process takes longer than 5 minutes.
+  #   restart_timeout = "5m"
+  # }
+
+  # provisioner "powershell" {
+  #   # Install utility packages
+  #   environment_vars = [
+  #     "Category=utilities",
+  #     "DriveLetter=${var.drive_letter}",
+  #     "PackagesDir=${var.packages_dir}"
+  #   ]
+  #   script = "src/powershell/install-category.ps1"
+  # }
+
+  # provisioner "windows-restart" {
+  #   # Wait a maximum of 5 minutes for Windows to restart.
+  #   # The build will fail if the restart process takes longer than 5 minutes.
+  #   restart_timeout = "5m"
+  # }
+
+  # provisioner "powershell" {
+  #   # Install vulnerability analysis packages
+  #   environment_vars = [
+  #     "Category=vulnerability-analysis",
+  #     "DriveLetter=${var.drive_letter}",
+  #     "PackagesDir=${var.packages_dir}"
+  #   ]
+  #   script = "src/powershell/install-category.ps1"
+  # }
+
+  # provisioner "windows-restart" {
+  #   # Wait a maximum of 5 minutes for Windows to restart.
+  #   # The build will fail if the restart process takes longer than 5 minutes.
+  #   restart_timeout = "5m"
+  # }
+
+  # provisioner "powershell" {
+  #   # Install web application packages
+  #   environment_vars = [
+  #     "Category=web-applications",
+  #     "DriveLetter=${var.drive_letter}",
+  #     "PackagesDir=${var.packages_dir}"
+  #   ]
+  #   script = "src/powershell/install-category.ps1"
+  # }
+
+  # provisioner "windows-restart" {
+  #   # Wait a maximum of 5 minutes for Windows to restart.
+  #   # The build will fail if the restart process takes longer than 5 minutes.
+  #   restart_timeout = "5m"
+  # }
+
+  # provisioner "powershell" {
+  #   # Install wordlist packages
+  #   environment_vars = [
+  #     "Category=wordlists",
+  #     "DriveLetter=${var.drive_letter}",
+  #     "PackagesDir=${var.packages_dir}"
+  #   ]
+  #   script = "src/powershell/install-category.ps1"
+  # }
```

Comment:
Can any of this stuff that is commented out be removed?

Reply:
I'm currently in the process of renaming the alot of deprecated and removed packages. I'm going to convert this PR to a draft as it is not ready

Reply:
@jsf9k this has been resolved in: [c26a7cc](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/c26a7cc1a45ba8feff464e7b5c094f1aca496b34)

Most of these lines have been deleted as the packages are now organized differently